src/diffusers/utils/pil_utils.py

```diff
@@ -0,0 +1,21 @@
+import PIL.Image
+import PIL.ImageOps
+from packaging import version
+
+
+if version.parse(version.parse(PIL.__version__).base_version) >= version.parse("9.1.0"):
+    PIL_INTERPOLATION = {
+        "linear": PIL.Image.Resampling.BILINEAR,
+        "bilinear": PIL.Image.Resampling.BILINEAR,
+        "bicubic": PIL.Image.Resampling.BICUBIC,
+        "lanczos": PIL.Image.Resampling.LANCZOS,
+        "nearest": PIL.Image.Resampling.NEAREST,
+    }
+else:
+    PIL_INTERPOLATION = {
+        "linear": PIL.Image.LINEAR,
+        "bilinear": PIL.Image.BILINEAR,
+        "bicubic": PIL.Image.BICUBIC,
+        "lanczos": PIL.Image.LANCZOS,
+        "nearest": PIL.Image.NEAREST,
+    }
```

Comment:
I'd prefer these to be symbols to prevent mistakes and allow autocompletion. Can we use `Enum`s  maybe?

Reply:
Makes sense!

Reply:
Actually, no this will lead to a weird API then because they are functions that are not used to compare anything. I don't want to add `if ...` comparisons everywhere in the code. Think it's actually fine as is. 

E.g. If I do:

```python
class PIL_INTERPOLATION(Enum): 
    LINEAR = PIL.Image.Resampling.BILINEAR
```

Then one **cannot** call the function with PIL_INTERPOLATION.LINEAR only with PIL_INTERPOLATION.LINEAR.value which is weird. Actually keen to leave as is here -> we have the same logic in `transformers` for the activation functions: https://github.com/huggingface/transformers/blob/4fb34de99e800689862f80cd618e6468fcb7b28b/src/transformers/activations.py#L152

Reply:
Wdyt @pcuenca ?

Reply:
Oh, that's weird, I thought it would work without `.value`. That's too bad, let's keep them as strings then.